.buildkite/pipeline.sh

```diff
@@ -24,4 +24,8 @@ echo "steps:
   - label: \":rust: Release image\"
     command: cd release && make build && make push
     branches: master
+
+  - label: \":rust: Database images\"
```

Comment:
These are not database images, but mssql images. misleading naming.

Reply:
fixed